### DISCUSSION TITLE
plus.frigate.video image reivew seems to per-camera now?

### DISCUSSION THREAD

**Discussion by edalquist** [blakeblackshear/frigate#14074] [Ask A Question]
Not sure when this started but when I go and review images on the plus.frigate site "Verify & Save" seems to go to the next untagged image for that camera and then when all the images for that camera are done I get back to the full images list. I then I have to find the next untagged image for another camera and repeat. If I remember right the site used to go through all untagged images across all cameras.

**Comment by NickM-27** [comment]
Yes, it appears this is the case and I believe it is because when new labels are supported the review will be by camera since typically different cameras support different labels. I believe the UI will be changing considerably to have filters and other stuff too.
